package.json

```diff
@@ -33,9 +33,10 @@
   "dependencies": {
     "@headlessui/react": "^1.7.18",
     "@heroicons/react": "^1.0.6",
-    "@polkadot/api": "15.8.1",
-    "@polkadot/api-contract": "15.8.1",
+    "@polkadot/api": "file:../polkadot-js-api/packages/api/build",
```

Comment:
To update once this https://github.com/polkadot-js/api/pull/6158 gets merged and released